v3/src/components/graph/adornments/movable-line/movable-line-adornment-model.ts

```diff
@@ -24,6 +57,12 @@ export const MovableLineInstance = types.model("MovableLineInstance", {
   },
   setPivot2(point: Point) {
     self.pivot2.set(point)
+  },
+  setVolatileIntercept(intercept?: number) {
+    self.dynamicIntercept = intercept
+  },
+  setVolatileSlope(slope?: number) {
+    self.dynamicSlope = slope
```

Comment:
You don't need two functions given that they're always called together:
```suggestion
  setVolatile(intercept?: number, slope?: number) {
    self.dynamicIntercept = intercept
    self.dynamicSlope = slope
```